python/lsst/afw/geom/transform/transform.cc

```diff
@@ -99,9 +99,9 @@ void declareTransform(py::module &mod, std::string const &fromName, std::string
      * of length 1 from being deleted */
     cls.def("_getJacobian", &Class::getJacobian);
 
-    declareMethodTemplates<GenericEndpoint, FromEndpoint, ToEndpoint>(cls);
-    declareMethodTemplates<Point2Endpoint, FromEndpoint, ToEndpoint>(cls);
-    declareMethodTemplates<SpherePointEndpoint, FromEndpoint, ToEndpoint>(cls);
+    declareMethodTemplates<FromEndpoint, ToEndpoint, GenericEndpoint>(cls);
```

Comment:
I find this form slightly less readable because it puts the most important information at the end, though I can see why you changed it.